index.js

```diff
@@ -1,569 +0,0 @@
-    //tried to create an inventory system for player and pc, but settled for "pc lite" due to time constraints. Looking at this function I built, there must be a better way to do this.
-    async startPc() {
-        console.log(`What do you want to do?`)
-        let input = await ask('Withdraw item, Deposit item, Toss item, or Log off\n>_');
-        sanitize(input);
-        input = input.trim();
-        input = input.split(' ');
-        let command = input[0];
-        let noun = input[input.length - 1];
-        if (command.includes('log off') || noun.includes('off')) {
-            console.log(`Got off the PC.`)
-            return start();
-        } else if (command.includes('withdraw')) {
-            console.log(`Pc inventory: ${this.pc}\nPlayer inventory: ${player.inventory}`);
-            let input = await ask(`What do you want to withdraw?\n`);
-            if (input.includes('potion')) {
-                if (bedRoom.pc.includes(`Potion x1`)) {
-                    player.inventory.push(`Potion x1`);
-                    bedRoom.pc.pop();
-                    console.log(`Withdrew potion.`)
-                    return this.startPc();
-                } else {
-                    console.log(`Nothing here to withdraw!`);
-                    return this.startPc();
-                }
-            } else {
-                console.log(`I didn't understand that command.`);
-                return this.startPc();
-            }
-        } else if (command.includes(`deposit`)) {
-            console.log(`Pc inventory: ${this.pc}\nPlayer inventory: ${player.inventory}`);
-            let input = await ask(`What do you want to deposit?\n>_`);
-            if (input.includes('potion')) {
-                if (player.inventory.includes(`Potion x1`)) {
-                    bedRoom.pc.push(`Potion x1`);
-                    player.inventory.pop();
-                    console.log(`Stored the potion via PC.`)
-                    return this.startPc();
-                } else {
-                    console.log(`Nothing here to deposit!`);
-                    return this.startPc();
-                }
-            } else {
-                console.log(`I didn't understand that command.`);
-                return this.startPc();
-            }
-        } else if (command.includes(`toss`)) {
-            console.log(player.inventory);
-            let input = await ask(`What do you want to toss?`);
-            if (input === 'potion') {
-                if (player.inventory.length >= 1) {
-                    player.inventory.pop();
-                    console.log(`Got rid of the potion.`);
-                    return this.startPc();
-                } else {
-                    console.log(`You don't have anything to toss away.`);
-                    return this.startPc();
-                }
-            }
-        } else {
-            console.log(`I didn't understand that command.`);
-            return this.startPc();
-        }
-    }
-}
```

Comment:
Next time, when submitting a project, I would recommend removing all unused code to streamline it.  Hold onto ideas like this `startPc` function in a separate functional branch, but never on your master, or at least not until you're ready to merge.  This is 63 lines of unused code;  it makes the program harder to read, and does nothing to extend its functionality.  Great idea, but consider who might be looking at your code, and if these extra aspects are serving their purpose (that is, creating a fully functional product!)